network/server/tchannelthrift/node/service_test.go

```diff
@@ -668,7 +668,7 @@ func TestServiceFetchTagged(t *testing.T) {
 
 	req, err := idx.NewRegexpQuery([]byte("foo"), []byte("b.*"))
 	require.NoError(t, err)
-	qry := index.Query{req}
+	qry := index.Query{Query: req}
```

Comment:
more involved change but could make index.Query a type alias if you wanted.

i.e.

```m3db/m3db/storage/index/types.go

import ".../m3ninx/idx"

type Query idx.Query
```

Thoughts?

Reply:
I like the idea, I'll test it out to see how involved it is

Reply:
I only made this change because my IDE was complaining at me lol

Reply:
I looked into this, but I think it's more trouble than it's worth because if we define `Query` as `type Query idx.Query` then `Query` won't inherit any of the methods of `idx.Query` and we'll need to type cast it whenever we need an `idx.Query`.

Reply:
Yeah fair. Happy to leave as is.